### PR TITLE
Productivity & Gizra

### PR DESCRIPTION
**Pull request by amitaibu**
@RachelBaram @IshaDakota <strike>Still need to add thumb</strike>, but already can be reviewed. I can't really tell if it's extreme or not. I think I lost the ability to sense it 😜

### CODE REVIEW

File: src/_posts/2017-04-19-productivity-gizra.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Internal Email - as Post
+tags:
+  - Office Culture
```
Comment (amitaibu) [inline src/_posts/2017-04-19-productivity-gizra.md]: not sure which tags

File: src/_posts/2017-04-19-productivity-gizra.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+permalink: "/content/internal-email/"
+layout: post
+image: "/assets/images/posts/internal-email/thumb.jpg"
+published: true
+description: ""
```
Comment (amitaibu) [inline src/_posts/2017-04-19-productivity-gizra.md]: need to fill this as-well

File: src/_posts/2017-04-19-productivity-gizra.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+---
+
+Below you will find an internal letter I've sent to to `all@gizra.com` a day before this blog post was published.
```
Comment (IshaDakota) [inline src/_posts/2017-04-19-productivity-gizra.md]: I wouldn't publish the all employee email here - could end of getting spam to all employees.

File: src/_posts/2017-04-19-productivity-gizra.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+I don't remember reading one that is about a mediocre state. That is, a state of something that is mostly working, but it's not optimal -- in fact it's a good few levels below it.
+
+It's pretty clear why it's not done, although the more I've been thinking about it, I came to realize that it's those mediocre state that most companies share. And that actually, sharing the below email could help others more than tooting our own horn.
+
+Now, don't get me wrong. If you ever heard me talk about Gizra, you know I think highly of us, and we have a long resume to prove it. But that only made writing this post even more exciting.  
+It's not even about putting myself out of my comfort zone (because I'm not). It's because a life long experience of doing things slightly different showed me it (usually) leads to good and surprising things. Sometimes it just blows in your face.
```
Comment (IshaDakota) [inline src/_posts/2017-04-19-productivity-gizra.md]: add line break or make part of previous paragraph (probably better).

Comment (IshaDakota) [inline src/_posts/2017-04-19-productivity-gizra.md]: "blows up in your face"

File: src/_posts/2017-04-19-productivity-gizra.md (hunk 3 above)
Comment (amitaibu) [inline src/_posts/2017-04-19-productivity-gizra.md]: AFAIK it's blocked from the outside (can you confirm that @bricel )

File: src/_posts/2017-04-19-productivity-gizra.md (hunk 2 above)
Comment (amitaibu) [inline src/_posts/2017-04-19-productivity-gizra.md]: @IshaDakota could you help me with this one please.

File: src/_posts/2017-04-19-productivity-gizra.md (hunk 1 above)
Comment (amitaibu) [inline src/_posts/2017-04-19-productivity-gizra.md]: @RachelBaram ^^

File: src/_posts/2017-04-19-productivity-gizra.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
+---
+title: Internal Email - as Post
+tags:
+  - Office Culture
+permalink: "/content/internal-email/"
+layout: post
+image: "/assets/images/posts/internal-email/thumb.jpg"
+published: true
+description: "At Gizra, having a new productivity system in place means that each employee can share in increasing the bottom line."
```
Comment (IshaDakota) [inline src/_posts/2017-04-19-productivity-gizra.md]: How's that?

File: src/_posts/2017-04-19-productivity-gizra.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+| 100 | Not exciting, but pays the bills |
+| 100 | They have chance with that product. Fine people |
+| 100  | Boring and annoying |
+| 90  | Thank god for having them
+| 90 | It's actually more fun than I thought it would |
+| 70 | Nice start |
+| 25 | Come on, how much do you expect to get for 25 hours?! |
+| 20 | I have mostly bad things to say about them |
+| 20 | Project X, unfortunately you'll see it also in the Non-billable |
+
```
Comment (liatsade) [inline src/_posts/2017-04-19-productivity-gizra.md]: Sorry for breaking in, but this is awesome and very entertaining!!! I enjoyed so much just reading what you wrote about each client...big like 😄

File: src/_posts/2017-04-19-productivity-gizra.md (hunk 5 above)
Comment (amitaibu) [inline src/_posts/2017-04-19-productivity-gizra.md]: I've changed it to a more click-baity description

File: src/_posts/2017-04-19-productivity-gizra.md (hunk 1 above)
Comment (amitaibu) [inline src/_posts/2017-04-19-productivity-gizra.md]: Never mind `- The Gizra Way` seems like the obvious choice.

**Comment by amitaibu** [comment]
Ready for review

**Comment by amitaibu** [comment]
@IshaDakota ^^

**Comment by IshaDakota** [comment]
Oh... didn't realize you were waiting for me. 

Yeah - publish that.

My only comment on the description is that those generally have some SEO value, and being click-baity sort of defeats that purpose.. But being click-baity has it's own value. Tom-a-to... Tom-ah-to

**Pull request merged**